src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="serch_bar-container serch_bar-condition"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <span class="input_icon">
+        <img src="../src/images/Search.svg" alt="finder-logo">
+      </span>
+      <label>
+        <input
+        id="input_big"
+        type="text"
+        class="input_big"
+        placeholder="Try &#34Los Angeles&#34"
+        >
+      </label>
+    </form>
+
+    <form
+      class="serch_bar-container-small serch_bar-condition"
+      action="#"
+      method="post"
+      data-qa="small"
+    >
+      <span class="input_icon-small">
+        <img src="../src/images/Search.svg" alt="finder-logo"
+          width="11"
+          height="11"
+        >
```

Comment:
```suggestion
        <img
          src="../src/images/Search.svg"
          alt="finder-logo"
          width="11"
          height="11"
        >
```

Reply:
Set width and height in the CSS, it is more clear and predictably

Reply:
done